tests/test_ec2uploadimg.py

```diff
@@ -1329,14 +1360,18 @@ def test_main_happy_path_snapOnly(
 
 
 @patch('ec2uploadimg.ec2upimg.EC2ImageUploader')
+@patch('ec2imgutils.ec2setup.EC2Setup.create_vpc_subnet')
 def test_main_happy_path_rootSwapMethod(
+    create_vpc_subnet_mock,
     EC2ImageUploader_mock,
     caplog
 ):
     ec2I = MagicMock()
     ec2I.create_image_use_root_swap.return_value = 'myAmi'
     EC2ImageUploader_mock.return_value = ec2I
 
+    create_vpc_subnet_mock.return_value = 'subnetVPCValue'
```

Comment:
can we add an assert in this test where we check `create_vpc_subnet` was called ?